scripts/gke/destroy-old-clusters.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 [-t <hours>] [-p <string>]" 1>&2; exit 1; }
+numreg='^[0-9]+$'
+
+while getopts ":t:p:" o; do
+    case "${o}" in
+        p)
+            p=${OPTARG}
```

Comment:
p identifier is Ok when its scope just a few lines of code, otherwise a more descriptive name is needed for readability sake.